apps/admin-gui/src/app/shared/components/dialogs/remove-service-from-resource-dialog/remove-service-from-resource-dialog.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { NotificatorService } from '@perun-web-apps/perun/services';
+import { TranslateService } from '@ngx-translate/core';
+import { ResourcesManagerService, Service } from '@perun-web-apps/perun/openapi';
+import { MatTableDataSource } from '@angular/material/table';
+
+export interface RemoveServiceFromResourceDialogData {
+  resourceId: number;
+  services: Service[]
+}
+
+@Component({
+  selector: 'app-perun-web-apps-remove-service-from-resource-dialog',
+  templateUrl: './remove-service-from-resource-dialog.component.html',
+  styleUrls: ['./remove-service-from-resource-dialog.component.scss']
+})
+export class RemoveServiceFromResourceDialogComponent implements OnInit {
+
+
+  constructor(public dialogRef: MatDialogRef<RemoveServiceFromResourceDialogComponent>,
+              @Inject(MAT_DIALOG_DATA) public data: RemoveServiceFromResourceDialogData,
+              private notificator: NotificatorService,
+              private translate: TranslateService,
+              private resourcesManager: ResourcesManagerService) {
+  }
+
+  loading: boolean;
```

Comment:
You probably wanted to use the loading when performing the removeServices action to hide the table and show the user that someting is doing, but forget to use it at all. Please add that ass well.